src/main/java/spoon/experimental/modelobs/EmptyModelChangeListener.java

```diff
@@ -33,6 +36,7 @@
 	@Override
 	public void onObjectUpdate(CtElement currentElement, CtRole role,
 			CtElement newValue, CtElement oldValue) {
+		this.changeCu(currentElement);
```

Comment:
Let's rename this type from EmptyModelChangeListener to DefaultModelChangeListener ... because it is no more empty
WDYT?

Reply:
Yes I did not plan to keep it like that ;)